packages/datetime2/test/isotest.mjs

```diff
@@ -31,7 +31,13 @@ describe("@blueprintjs/datetime2 isomorphic rendering", () => {
             DateRangePicker3: {},
         },
         {
-            excludedSymbols: ["DateInput2", "DateInput2MigrationUtils", "DateRangeInput2", "TimezoneSelect"],
+            excludedSymbols: [
+                "DateInput2",
+                "DateInput2MigrationUtils",
+                "DateRangeInput2",
+                "MonthAndYear",
```

Comment:
huh, do you know what this was needed for?

Reply:
The test was failing and `MonthAndYear` is also excluded in the same place in `/datetime/test/isotest.mjs`